tests/conftest.py

```diff
@@ -19,3 +19,19 @@ def teardown():
     request.addfinalizer(teardown)
 
     return flaskr.app
+
+
+@pytest.fixture(scope='function')
+def base_url(live_server):
+    return live_server.url()
+
+
+@pytest.fixture(scope='function')
+def sensitive_url(request):
+    return False
+
+
+@pytest.fixture(scope='function', autouse=True)
```

Comment:
Actually I have another question. Can you explain this? Is it the case that pytest-selenium will no longer attempt to verify the base_url, and you need to ask it to do so? Also, will this verify it for every test, as opposed to just once at the beginning of all tests?